### PR TITLE
2499 activate wrap text in workbook

### PR DESCRIPTION
**Pull request by sambodeme**
For details, see #2499 

## PR checklist: submitters

- [x]   Link to an issue if possible. If there’s no issue, describe what your branch does. Even if there is an issue, a brief description in the PR is still useful.
- [ ]   List any special steps reviewers have to follow to test the PR. For example, adding a local environment variable, creating a local test file, etc.
- [ ]   For extra credit, submit a screen recording like [this one](https://github.com/GSA-TTS/FAC/pull/1821).
- [ ]   Make sure you’ve merged `main` into your branch shortly before creating the PR. (You should also be merging `main` into your branch regularly during development.)
- [ ]   Make sure you’ve accounted for any migrations. When you’re about to create the PR, bring up the application locally and then run `git status | grep migrations`. If there are any results, you probably need to add them to the branch for the PR. Your PR should have only **one** new migration file for each of the component apps, except in rare circumstances; you may need to delete some and re-run `python manage.py makemigrations` to reduce the number to one. (Also, unless in exceptional circumstances, your PR should not delete any migration files.)
- [ ]   Make sure that whatever feature you’re adding has tests that cover the feature. This includes test coverage to make sure that the previous workflow still works, if applicable.
- [ ]   Do manual testing locally. Our tests are not good enough yet to allow us to skip this step. If that’s not applicable for some reason, check this box.
- [ ]   Verify that no Git surgery was necessary, or, if it was necessary at any point, repeat the testing after it’s finished.
- [ ]   Once a PR is merged, keep an eye on it until it’s deployed to dev, and do enough testing on dev to verify that it deployed successfully, the feature works as expected, and the happy path for the broad feature area (such as submission) still works.

## PR checklist: reviewers

- [ ]   Pull the branch to your local environment and run `make docker-clean; make docker-first-run && docker compose up`; then run `docker compose exec web /bin/bash -c "python manage.py test"`
- [ ]   Manually test out the changes locally, or check this box to verify that it wasn’t applicable in this case.
- [ ]   Check that the PR has appropriate tests. Look out for changes in HTML/JS/JSON Schema logic that may need to be captured in Python tests even though the logic isn’t in Python.
- [ ]   Verify that no Git surgery is necessary at any point (such as during a merge party), or, if it was, repeat the testing after it’s finished.
        
The larger the PR, the stricter we should be about these points.

### CODE REVIEW

File: backend/schemas/scripts/render.py [hunk 1]
```diff
@@ -499,6 +501,46 @@ def set_open_range_and_single_cell_widths(ws, sheet):
...
+        if ws.title in ("Form", "AdditionalNotes"):
+            for named_range_name in wb.defined_names:
+                activate_wraptext_for_named_range(wb, ws, named_range_name)
+
+
+def activate_wraptext_for_named_range(wb, ws, named_range_name):
+    """Activate wrapText for all cells within a named range in the worksheet."""
+    if named_range_name in wb.defined_names:
+        named_range = wb.defined_names[named_range_name]
+        if named_range.attr_text.startswith(
```
Comment (tadhg-ohiggins) [inline backend/schemas/scripts/render.py:535]: Where are these string values, with the single quotation marks and the exclamation marks, coming from?

**Review by tadhg-ohiggins — APPROVED** [review]

File: backend/schemas/scripts/render.py (hunk 1 above)
Comment (sambodeme) [reply backend/schemas/scripts/render.py:535]: These are sheets named in the workbooks. For instance, for audit findings text, it is defined [here](https://github.com/GSA-TTS/FAC/pull/2518/files#diff-a5302ef12de61ad2603bca96749d4e35a95169b68e02115ee16cac8a4ef1df3bR5) and passthrough [here](https://github.com/GSA-TTS/FAC/pull/2518/files#diff-a5302ef12de61ad2603bca96749d4e35a95169b68e02115ee16cac8a4ef1df3bR104). Similarly, for CAP, it is[ here](https://github.com/GSA-TTS/FAC/pull/2518/files#diff-a5302ef12de61ad2603bca96749d4e35a95169b68e02115ee16cac8a4ef1df3bR104)

Comment (sambodeme) [reply backend/schemas/scripts/render.py:535]: Thinking of a way to mitigate situations where the sheet name gets changed? We could store these specific sheet names in Sheets.libsonnet and read them from there.

**Review by jadudm — APPROVED** [review]
Looks good.

**Pull request merged**